### PR TITLE
booking, header, login, payment and reg styling.

### PR DESCRIPTION
**Pull request by TaiwoAB**

### CODE REVIEW

File: app/views/registration.scala.html [hunk 1]
```diff
@@ -1,15 +1,62 @@
...
+                        background-color: gainsboro;"></i>
+                    @helper.inputPassword(form("password"), '_label -> "", '_showConstraints -> false, 'id->"regpasswordform", 'placeholder->"Password")
+                </div>
+
+
+
+            </center>
+
+            <button type="submit" class="btn mb-3 col-9 mx-auto reg_btn" style="background-color: navy;
+                color: white; display: block; width: 100% ; margin-top: 48px;">Sign In</button>
```
Comment (DannyPennington) [inline app/views/registration.scala.html]:
```suggestion
                color: white; display: block; width: 100% ; margin-top: 48px;">Sign Up</button>
```

**Review by DannyPennington — CHANGES_REQUESTED** [review]
Looking good, just a small text change on the registration button

**Review by DannyPennington — APPROVED** [review]
Nice one

**Review by AIbrahimQA — APPROVED** [review]
Looks good

**Pull request merged**